vector/src/main/res/layout/item_other_session.xml

```diff
@@ -67,19 +67,30 @@
         android:layout_height="wrap_content"
         android:layout_marginTop="2dp"
         android:drawablePadding="8dp"
-        app:layout_constraintBottom_toBottomOf="@id/otherSessionDeviceTypeImageView"
         app:layout_constraintEnd_toEndOf="@id/otherSessionNameTextView"
         app:layout_constraintStart_toStartOf="@id/otherSessionNameTextView"
         app:layout_constraintTop_toBottomOf="@id/otherSessionNameTextView"
         tools:text="@string/device_manager_verification_status_verified" />
 
+    <TextView
```

Comment:
I think we should update some constraints on this layout:
- For the view with id `otherSessionItemBackground` we should replace `app:layout_constraintBottom_toBottomOf="@id/otherSessionVerificationStatusImageView"` by `app:layout_constraintBottom_toBottomOf="@id/otherSessionIpAddressTextView"`
- For the view with id `otherSessionNameTextView` we should replace `app:layout_constraintTop_toTopOf="@id/otherSessionDeviceTypeImageView"` by `app:layout_constraintTop_toTopOf="@id/otherSessionItemBackground"`

Reply:
Good catch, done.